README.md

```diff
@@ -24,6 +24,7 @@ The following packages are also recommended, but you can configure `setup.py` to
 instead of them:
 
  * libreoffice
+ * imagemagick
```

Comment:
Pri commanline `imagemagick` uz netreba hackovat to nastavenie v `/etc`? Pls prever. Ak nie tak ok. Ak ano, tak napis do README navod, co treba spravit.

Reply:
:ok: